### ISSUE TITLE
`Unexpected token` in RegExp statement

### ISSUE THREAD

**Issue by chriswong** [babel/babel-eslint#161]
```javascript
if (Boolean) {
}

/foo/.test('bar') ? true : false;
```

>  4:0  error  Unexpected token

```javascript
if (Boolean) {
};

/foo/.test('bar') ? true : false;
```

> 4:0   error  Unnecessary semicolon

**Comment by hzoo** [comment]
Well if it's a parsing error then probably a bablyon issue? Oh is this the same as https://github.com/babel/babel/issues/1909?

**Comment by sebmck** [comment]
Yeah it is.

**Comment by hzoo** [comment]
Looks like it was fixed in acorn already? Added ` || prevType === tt.parenR` to https://github.com/babel/babel/blob/e2d673dafa8b88604e2738916e0235ffc0852de8/packages/babylon/src/tokenizer/index.js#L789 

https://github.com/marijnh/acorn/commit/791daa65ce05c6621ff2d7660c8cb6bd38b90be5

**Comment by sebmck** [comment]
Oh shit, yeah, didn't notice that. Want to PR it?

**Comment by hzoo** [comment]
Ok sure! - although looks like the tests are fixtures?

**Comment by sebmck** [comment]
Ah yeah. You can just create a folder and put an `actual.js` there. When you run the test suite if an `expected.json` isn't there it'll create one. You can omit that from the PR if you don't want to get a local Babel development environment going.

**Comment by hzoo** [comment]
Oh ok that's cool! I'l go ahead and do it. Do I just make a folder in `fixtures/core/uncategorized/` with the next number?

**Comment by sebmck** [comment]
Try and categorise it if you can (feel free to make a new category). All of those tests are just numbers and in the uncategorised folder because they were imported from other test suites that had weird formats.

**Comment by hzoo** [comment]
Ok I just made a `categorized` folder since it seemed like it wouldn't created the `expected.json` without a subfolder
